commerce-cloud-code/app_yourshophere/cartridge/controllers/Account.js

```diff
@@ -59,4 +59,19 @@ server.post('DeleteAddress', server.middleware.https, userLoggedIn.validateLogge
     return next();
 });
 
+server.get('Profile', server.middleware.https, userLoggedIn.validateLoggedIn, (req, res, next) => {
```

Comment:
I don't if this is maybe too "extreme" but in the end these are just pages, right? So why not lnk their Page-Show urls?

Reply:
Yes, that would certainly work, it would solve the linking challenges as well.